mendeleev/econf.py

```diff
@@ -90,12 +106,12 @@ def conf(self, value):
             raise ValueError(f"<conf> should be str or dict, got {type(value)}")
 
     @property
-    def atomre(self):
+    def atomre(self) -> str:
```

Comment:
This is a compiled regular expression object and not a string.

Reply:
Thanks, I just updated the type hint